tests/testthat/test-renv_helpers.R

```diff
@@ -116,6 +116,14 @@ testthat::test_that("testing renv_helpers", {
     })
     testthat::expect_equal(call, test_call)
     testthat::expect_match(warns, "has the unsupported remote type")
+
+    warns <- testthat::capture_warnings({
+      call <- renv2nix(tmpf, return_rix_call = TRUE, ide = "rstudio")
+    })
+    test_call$ide <- "rstudio"
+    testthat::expect_equal(call, test_call)
+    testthat::expect_match(warns, "has the unsupported remote type")
```

Comment:
why not expect_warning?

https://testthat.r-lib.org/reference/expect_error.html

Reply:
It's a pattern I got into working on another project where I'm regularly testing things that throw multiple warnings, `capture_warnings()` let's me test I've got the right set of warnings. In this case `expect_warning()` is indeed better.